Gemfile.lock

```diff
@@ -33,7 +33,7 @@ GIT
 GEM
   remote: https://rubygems.org/
   specs:
-    aasm (5.0.8)
+    aasm (5.1.1)
       concurrent-ruby (~> 1.0)
```

Comment:
Updated aasm as I was aware they'd changed their transaction handling, and didn't wanting to waste time with a fix which could break immediately.